install_config/install/docker_registry.adoc

```diff
@@ -1013,6 +1013,7 @@ storage:
     readonly:
       enabled: false
 ----
+<1> This entry is *mandatory* for image pruning to work properly.
```

Comment:
Can you also explain, what it actually does (enables removal of data on registry's storage through API)?